### PR TITLE
Release candidate for Release v0.31.0

### PR DESCRIPTION
**Pull request by mkistler**
This PR is the release candidate for SDK release v0.31.0.

See the Changelog.md for the changes included in this release.

api-definitions git hash: 550d10e	
watson-swagger-codgen git hash: edf43c30

### CODE REVIEW

**Comment by mkistler** [comment]
All Travis CI failures appear to be service timeouts.

**Comment by AnthonyOliveri** [comment]
@mkistler Is there anything you want me to review here, or has it all been covered by previous PR reviews?

**Comment by AnthonyOliveri** [comment]
The Travis tests passed! 🎉

**Comment by mkistler** [comment]
Nice catch Anthony!  I'll tag you for a review ... we just have to hold off on merging.

File: Source/AssistantV1/Assistant.swift [hunk 1]
```diff
@@ -1935,6 +1939,78 @@ public class Assistant {
         }
     }
 
+    /**
+     List entity mentions.
+
+     List mentions for a contextual entity. An entity mention is an occurrence of a contextual entity in the context of
+     an intent user input example.
```
[the remark this answers is not in the document]
Comment (mkistler) [reply Source/AssistantV1/Assistant.swift]: That comes straight from the API doc.  @rrberry is this description correct?

[the remark this answers is not in the document]
Comment (rrberry) [reply Source/AssistantV1/Assistant.swift]: Yes. A contextual entity is defined as an entity that is mentioned in the text of an intent user example. An entity mention shows you the intent example and where the entity text occurs in that example. See also https://console.bluemix.net/docs/services/conversation/entities.html#defining-contextual-entities

**Pull request merged**